l10n_fr_ecotaxe/models/product_product.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2021 Camptocamp
+#   @author Silvio Gregorini <silvio.gregorini@camptocamp.com>
+# Copyright 2023 Akretion (http://www.akretion.com)
+# #   @author Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+from odoo.osv import expression
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    additional_ecotaxe_line_product_ids = fields.One2many(
+        "ecotaxe.line.product",
+        "product_id",
+        string="Additional ecotaxe lines",
+        copy=True,
+        domain="[('id', 'not in', ecotaxe_line_product_ids)]",
+    )
+    all_ecotaxe_line_product_ids = fields.One2many(
+        "ecotaxe.line.product",
+        compute="_compute_all_ecotaxe_line_product_ids",
+        search="_search_all_ecotaxe_line_product_ids",
+        string="Additional ecotaxe lines",
```

Comment:
```suggestion
        string="All ecotaxe lines",
```
To avoid Warning : `Two fields (all_ecotaxe_line_product_ids, additional_ecotaxe_line_product_ids) of product.product() have the same label: Additional ecotaxe lines.`